<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Credit_calc</class>
 <widget class="QWidget" name="Credit_calc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>403</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Расчитать</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_anu">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>107</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Аннуитетный</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QCheckBox" name="checkBox_difer">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>169</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Дифференцированный</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="label_sum_cred">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>98</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма кредита</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_term">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>87</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Срок кредита</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_procent">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Процентная ставка</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_proc">
   <property name="geometry">
    <rect>
     <x>226</x>
     <y>98</y>
     <width>12</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_rub">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>23</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Руб</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_month_pr">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>137</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ежемесячный платёж</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_porc_pr">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Начисленные проценты</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dolg_proc">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>107</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Долг + проценты</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_calc_month">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>220</y>
     <width>410</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0.00</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_calc_proc">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>410</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0.00</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_calc_sum">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>410</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0.00</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>222</x>
     <y>54</y>
     <width>103</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Лет</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Месяцев</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_type">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>174</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Тип ежемесячного платежа</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textEdit_sum">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>13</y>
     <width>168</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="textEdit_term">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>51</y>
     <width>80</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="textEdit_procent">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>80</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
